services/chatbot/src/chatbot/chat_api.py

```diff
@@ -34,19 +37,30 @@ async def init():
     await store_api_key(session_id, openai_api_key)
     return jsonify({"message": "Initialized"}), 200
 
+@chat_bp.route("/model", methods=["POST"])
```

Comment:
Should be changeable only on deployment. Remove this method. Only source should be env.